tools/msg.c

```diff
@@ -228,11 +228,13 @@ bool bl_msg_yaml_parse(FILE *file, union bl_msg_data *msg)
 		msg->start.frequency  = bl_msg__yaml_read_unsigned(file, "Frequency",  &ok);
 		msg->start.oversample = bl_msg__yaml_read_unsigned(file, "Oversample", &ok);
 		msg->start.src_mask   = bl_msg__yaml_read_hex(file, "Source Mask",     &ok);
+		msg->start.led_mask   = bl_msg__yaml_read_hex(file, "LED Mask",     &ok);
 		break;
 
 	case BL_MSG_SAMPLE_DATA16:
 		msg->sample_data.channel = bl_msg__yaml_read_unsigned(file, "Channel", &ok);
 		msg->sample_data.count   = bl_msg__yaml_read_unsigned(file, "Count",   &ok);
+		msg->sample_data.src_led = bl_msg__yaml_read_unsigned(file, "Source: LED",   &ok);
```

Comment:
This needs to go above "Count", because the parser ordering needs to match the print ordering in `bl_msg_yaml_print`.

Reply:
+1